samples/testing-frameworks/appium/client-side/python/README.md

```diff
@@ -127,7 +151,7 @@ To run iOS native app tests additional parameter is required to be provided:
 * **bundleId** - this is your application's unique name
 
 ```bash
-$ python run-test.py -k xYY5...PeOA6 -s /tmp/screens/ -p "iOS" -r `date +%R` -a "127314812" --bundle_id "com.bitbar.testdroid.BitbarIOSSample" -t bitbar_ios  
+$ python run-test.py -k xYY5...PeOA6 -s ./screenshots -p "iOS" -r `date +%R` -a "127314812" --bundle_id "com.bitbar.testdroid.BitbarIOSSample" -t bitbar_ios  
```

Comment:
to make it work we need to add `screenshots` directory to the repo. To make git keep that empty directory in repo, please add there `.gitkeep` file